api/controllers/v1/jobs.ts

```diff
@@ -201,6 +228,18 @@ export const extractUrlFromMessage = (fullDocument): string[] => {
   return urls.map((url) => url.replace(/([^:]\/)\/+/g, '$1'));
 };
 
+async function prepGithubComment(fullDocument: Job, jobUrl: string, isUpdate = false): Promise<string> {
+  if (isUpdate) {
+    return `\n* job log: [${fullDocument.payload.newHead}](${jobUrl})`;
+  }
+  const urls = extractUrlFromMessage(fullDocument);
+  let stagingUrl = '';
+  if (urls.length > 0) {
+    stagingUrl = urls[urls.length - 1];
+  }
+  return `✨ Staging URL: [${stagingUrl}](${stagingUrl})\n\n#### 🪵 Logs\n\n* job log: [${fullDocument.payload.newHead}](${jobUrl})`;
```

Comment:
Nice emoji 😎

Reply:
lol, was allison :) she's a queen of style

Reply:
Awww